crates/notion-core/src/distro/error.rs

```diff
@@ -1,27 +1,69 @@
 //! Provides error types for the installer tools.
 
+use archive::HttpError;
 use notion_fail::{ExitCode, NotionFail};
 
 use failure;
+use reqwest::StatusCode;
+use std::fmt;
+
+// Once Issue #173 is implemented, we can use the ToolSpec struct to differentiate tools
```

Comment:
It'd be nice to use a standard syntax for issue comments, esp for greppability. We've been using:

```rust
// ISSUE #173: Once it's implemented we can use the ToolSpec struct etc etc
```